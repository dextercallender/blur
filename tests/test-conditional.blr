void void() {
    int a;
    int b;
    a = 1;
    b = 9;

    if(a<b) {
	print("check 1", "%s");
    };

    if(a>b) {
	print("should not print", "%s");
    };
    else {
	print("check 2 no print", "%s");
    };
}
