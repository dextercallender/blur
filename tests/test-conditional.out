check 1check2
